php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -416,6 +424,7 @@ class NDB_BVL_Instrument extends NDB_Page
                 $formArray['tableID'] = "instrument";
             }
 
+            $formArray['lang'] = $_REQUEST["lang"] ? $_REQUEST["lang"] : "en-ca";
```

Comment:
is there an isset() missing around the first part of the ternary? Can this just be a null coalesce (`??`) or elvis operator (`?:`)?